src/99-appendices/03-hed.md

```diff
@@ -167,16 +167,16 @@ This allows for a proper validation of the HED annotations
 (for example using the `bids-validator`).
 
 Example: The following `dataset_description.json` file specifies that the
-[`HED8.0.0.xml`](https://github.com/hed-standard/hed-specification/tree/master/hedxml/HED8.0.0.xml)
+[`HED8.1.0.xml`](https://github.com/hed-standard/hed-specification/tree/master/hedxml/HED8.1.0.xml)
 file from the `hedxml` directory of the
 [`hed-specification`](https://github.com/hed-standard/hed-specification)
 repository on GitHub should be used to validate the study event annotations.
 
 ```JSON
 {
   "Name": "A great experiment",
-  "BIDSVersion": "1.6.0",
-  "HEDVersion": "8.0.0"
+  "BIDSVersion": "1.7.0",
```

Comment:
Since we're changing so much else of this file, is there a reason _not_ to change it? We recommend taggers use the latest version of the base schema, so having the documentation match the latest version seems reasonable. It's not like we're making a PR just for this one change.

Reply:
It is true that 8.0.0 supports everything from the perspective of the person annotating and it is a nice round number. I don't have a strong feeling either way on this.